node-w3capi/index.js

```diff
@@ -72,12 +75,38 @@ const html = renderHTML(data.users, data.orgs, {
 fs.writeFileSync(filenames.html.participants, html);
 spinner.succeed("Generated HTML")
 
+const nonSingularOrgs = data.orgs.filter(org => org.orgUsers.length > 1)
+
 spinner.start("🤖 Generating markdown")
-const markdown = renderMarkdown(data.users, data.orgs, {
-  name: "W3C Solid Community Group",
-  description: "test",
-});
-fs.writeFileSync(filenames.md.affiliations, markdown);
+  const markdown = tablemark(nonSingularOrgs
+    .map(o => ({
+      name: o.name,
+      members: o.orgUsers.length,
+      default: o.orgUsers[0].name,
+      designated: ''
+    }))
+  )
+fs.writeFileSync(filenames.md.default, markdown);
 spinner.succeed("Generated markdown")
 
+if (localDataExists(filenames.md)) {
+  spinner.start("🤖 Generating eligible voters list")
+  const representativesTable = fs.readFileSync(filenames.md.designated, 'utf-8')
+    .split('\n').slice(2).filter(Boolean)
+
+  const orgRepresentatives = representativesTable.map(row => {
+    const fallback = row.split('|')[3].trim()
+    const designated = row.split('|')[4].trim()
+    return designated || fallback
+  })
```

Comment:
Could we avoid this type of hacking? I'm sure you're doing it right for most intents and purposes but I find this kind of data slicing brittle.

Do we need to change the input format? Can we benefit from something simpler / more structured? JSON array? CSV? We should run a well-tested/known function to get the column or whatever.

Reply:
Since we already requested editing a markdown table we probably shouldn't change it now to JSON.
We could also search for an existing module that parses markdown tables but when I used one for serializing markdown tables @VirginiaBalseiro didn't like that approach.

While it ain't pretty it works and it seems that not many people actually care that much about this part of the process, at least this election so as long as it does the job I wouldn't worry any more than needed.